retail/interactive-tutorial/RetailSearch.Samples/SearchWithPageSizeSample.cs

```diff
@@ -90,6 +92,10 @@ public IEnumerable<SearchResponse> Search(string projectId)
             }
         }
 
+        // PASTE CALL WITH NEXT PAGE TOKEN HERE:
+
+        // PASTE CALL WITH OFFSET HERE:
+
```

Comment:
Where are they suppose to get this code from?

What happens to users that are looking at these samples in documentation and are not following a tutorial?
I remember there were a few samples showing how to do pagination already, so maybe this one can remain about page size only? And they can all come together on the tutorial itself?
Or you can show how to set page_size in one of the pagination samples themselves.

But I think this text here is very confusing for anyone that's not following the tutorial. These samples should be clear enough standalone samples.

And also, if this is going to stay, which I really think it shouldn't, please do not write in all caps.

Reply:
Comments indicate where to paste code during tutorial steps.
In other languages we have 1 sample in which we paste code. We have come to the conclusion that it will be better if this sample will be the same. 
Or maybe it would be better to create another one in which we will paste the code?

Reply:
But comments do not indicate which code you are referring to, that's only clear if you are following the tutorial. But these samples are to be made available through normal documentation as well, and are also here in GitHub were users can attempt to use them on their own. There's absolutetly no context for what these instruction comments mean for anyone except maybe for those who happen to be following the tutorials. Otherwise this is very confusing.

Reply:
Got it. But could we add fields to `searchRequest` and arguments to `GetSearchRequest `method?

Reply:
Yes, and actually you can add a description for those new fields as part of the XML doc of the method saying something like (or whatever wording you find appropiate or is similar in the tutorial)

```charp
/// <param name="offset">The offset from which the search will start</param>
/// <param name="nextPageToken">The next page token you obtained on a previous call which will allow to obtain the next page of results.</param>
```

Reply:
done